### ISSUE TITLE
NameAllocator isn't used when generating `encodeSize()` in Kotlin

### ISSUE THREAD

**Issue by JGulbronson** [square/wire#723]
The NameAllocator isn't used when generating the `encodeSize()` function, resulting in the following in the `FileDescriptorProto`:
```kotlin
  override fun encodedSize(value: FileDescriptorProto): Int = ProtoAdapter.STRING.encodedSizeWithTag(1, value.name) +
                ProtoAdapter.STRING.encodedSizeWithTag(2, value.package) +
                ProtoAdapter.STRING.asRepeated().encodedSizeWithTag(3, value.dependency) +
```

NameAllocator _is_ used when generating the field name, resulting in `package_`, but the resulting Kotlin code using `value.package` results in an error saying "Expecting an element".

**Comment by Egorand** [comment]
@mc-lovin wanna take a look?

**Comment by Egorand** [comment]
Fixed by #727
